packages/php-config-printer/src/NodeFactory/CommonNodeFactory.php

```diff
@@ -45,8 +45,8 @@ public function createConstFetch(string $className, string $constantName): Class
         return new ClassConstFetch(new FullyQualified($className), $constantName);
     }
 
-    public function createFalse(): ConstFetch
+    public function createBoolConstFetch(bool $value): ConstFetch
     {
-        return new ConstFetch(new Name('false'));
+        return new ConstFetch(new Name($value ? 'true' : 'false'));
```

Comment:
This opens door for false positive in some cases, e.g. `createBoolConstFetch('')` vs `createBoolConstFetch((bool) 'false')`. I would wonder how to make `null` with this method too.

Could you keep it as closed as previous method? This way:

```php
public function createFalse(): ConstFetch
public function createTrue(): ConstFetch
```

We can avoid some WTFs and improve re-use in other places.

Reply:
Changed.